### PR TITLE
Deploy using Kamal

### PR DESCRIPTION
**Pull request by EmilioCristalli**
~⚠️  depends on https://github.com/cedarcode/webauthn-2fa-rails-demo/pull/27 ⚠️~

### CODE REVIEW

File: .kamal/hooks/docker-setup.sample
Comment (santiagorodriguez96) [inline .kamal/hooks/docker-setup.sample:1]: Should we remove these samples?

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+# Make sure RUBY_VERSION matches the Ruby version in .ruby-version
+ARG RUBY_VERSION=3.3.5
+FROM docker.io/library/ruby:$RUBY_VERSION-slim AS base
+
+# Rails app lives here
+WORKDIR /rails
+
+# Install base packages
+RUN apt-get update -qq && \
+    apt-get install --no-install-recommends -y curl libjemalloc2 sqlite3 && \
```
Comment (santiagorodriguez96) [inline Dockerfile:16]: We should be able to remove the `sqlite3` package, right?

**Review by santiagorodriguez96 — COMMENTED** [review]
Left a couple of comments to address 🙂

File: .kamal/hooks/docker-setup.sample
Comment (EmilioCristalli) [reply .kamal/hooks/docker-setup.sample:1]: 👍  [e8b47b3](https://github.com/cedarcode/webauthn-2fa-rails-demo/pull/28/commits/e8b47b348ed9b8f45d37299e060ada401863eb89)

**Review by santiagorodriguez96 — APPROVED** [review]
LGTM

**Pull request merged**